gno.land/pkg/gnoland/mock_test.go

```diff
@@ -113,6 +118,82 @@ func (m *mockVMKeeper) CommitGnoTransactionStore(ctx sdk.Context) {
+	return nil, nil
+}
+
+func (m *mockBankKeeper) AddCoins(ctx sdk.Context, addr crypto.Address, amt std.Coins) (std.Coins, error) {
+	return nil, nil
+}
+
+func (m *mockBankKeeper) InitGenesis(ctx sdk.Context, data bank.GenesisState)     {}
+func (m *mockBankKeeper) GetParams(ctx sdk.Context) bank.Params                   { return bank.Params{} }
+func (m *mockBankKeeper) GetCoins(ctx sdk.Context, addr crypto.Address) std.Coins { return nil }
+func (m *mockBankKeeper) SetCoins(ctx sdk.Context, addr crypto.Address, amt std.Coins) error {
+	return nil
+}
+
+func (m *mockBankKeeper) HasCoins(ctx sdk.Context, addr crypto.Address, amt std.Coins) bool {
+	return true
+}
+
+type mockAuthKeeper struct{}
+
+func (m *mockAuthKeeper) NewAccountWithAddress(ctx sdk.Context, addr crypto.Address) std.Account {
+	return nil
+}
+func (m *mockAuthKeeper) GetAccount(ctx sdk.Context, addr crypto.Address) std.Account     { return nil }
+func (m *mockAuthKeeper) GetAllAccounts(ctx sdk.Context) []std.Account                    { return nil }
+func (m *mockAuthKeeper) SetAccount(ctx sdk.Context, acc std.Account)                     {}
+func (m *mockAuthKeeper) IterateAccounts(ctx sdk.Context, process func(std.Account) bool) {}
+func (m *mockAuthKeeper) InitGenesis(ctx sdk.Context, data auth.GenesisState)             {}
+func (m *mockAuthKeeper) GetParams(ctx sdk.Context) auth.Params                           { return auth.Params{} }
+
+type mockParamsKeeper struct{}
+
+func (m *mockParamsKeeper) GetString(ctx sdk.Context, key string, ptr *string)    {}
+func (m *mockParamsKeeper) GetInt64(ctx sdk.Context, key string, ptr *int64)      {}
+func (m *mockParamsKeeper) GetUint64(ctx sdk.Context, key string, ptr *uint64)    {}
+func (m *mockParamsKeeper) GetBool(ctx sdk.Context, key string, ptr *bool)        {}
+func (m *mockParamsKeeper) GetBytes(ctx sdk.Context, key string, ptr *[]byte)     {}
+func (m *mockParamsKeeper) GetStrings(ctx sdk.Context, key string, ptr *[]string) {}
+
+func (m *mockParamsKeeper) SetString(ctx sdk.Context, key string, value string)    {}
+func (m *mockParamsKeeper) SetInt64(ctx sdk.Context, key string, value int64)      {}
+func (m *mockParamsKeeper) SetUint64(ctx sdk.Context, key string, value uint64)    {}
+func (m *mockParamsKeeper) SetBool(ctx sdk.Context, key string, value bool)        {}
+func (m *mockParamsKeeper) SetBytes(ctx sdk.Context, key string, value []byte)     {}
+func (m *mockParamsKeeper) SetStrings(ctx sdk.Context, key string, value []string) {}
+
+func (m *mockParamsKeeper) Has(ctx sdk.Context, key string) bool             { return false }
+func (m *mockParamsKeeper) GetRaw(ctx sdk.Context, key string) []byte        { return nil }
+func (m *mockParamsKeeper) SetRaw(ctx sdk.Context, key string, value []byte) {}
+
+func (m *mockParamsKeeper) GetStruct(ctx sdk.Context, key string, strctPtr interface{}) {}
+func (m *mockParamsKeeper) SetStruct(ctx sdk.Context, key string, strct interface{})    {}
```

Comment:
```suggestion
func (m *mockParamsKeeper) GetAny(ctx sdk.Context, key string, strctPtr interface{}) {}
func (m *mockParamsKeeper) SetAny(ctx sdk.Context, key string, strct interface{})    {}
```

Struct is fine to me; but this is a potential alternative.

Reply:
I actually added GetAny, but it doesn't work with structs. Only primitives.
Keeper params are typed, so SetStruct() works well, but VM keeper RealmParams (say) are arbitrary user-given, and SetStruct() wouldn't be appropriate for user params.